plugin/upstream_generator/evaluator/file_evaluator.go

```diff
@@ -0,0 +1,45 @@
+package evaluator
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/goto/salt/log"
+
+	"github.com/goto/optimus/internal/errors"
+)
+
+type fileEvaluator struct {
```

Comment:
why don't we export it? asking because the treatment on `newFileEvaluator` seems like it's as if exported

Reply:
same reason with this https://github.com/goto/optimus/pull/160#discussion_r1356123821